modules/Intro/Intro.Rmd

```diff
@@ -397,7 +397,7 @@ You can just copy and paste the below code into your console - we'll explain wha
 ```{r, packageInstall, eval=FALSE}
 install.packages("remotes")
 library(remotes)
-install_github("muschellij2/jhur")
+remotes::install_github("jhudsl/jhur")
```

Comment:
Shouldn't need `remotes::`

Reply:
right! sorry I missed that- was trying to just replace the name :P and missed that when I tried to resolve the conflict